tables/adverse-events/aet04.qmd

```diff
@@ -491,6 +502,15 @@ result <- prune_table(raw_table, keep_content_rows(my_row_condition(row_conditio
 result
 ```
 
+```{r save_aet04, echo=FALSE, message=FALSE}
+save(result_v1,
+  result_v2,
+  result_v3,
+  result_v4,
+  file = paste0(main_pkg_dir, "inst/extdata/aet04.Rdata")
```

Comment:
Rdata is used to save multiple R objects, while Rds is used to save a single R object

Reply:
gitignore is a good point, buildignore too. I will add it now. The logic behind using inst/extdata was related to the fact that the user should not access it in anyway but saving it in the test folder makes more sense

Reply:
actually I checked and buildignore (ofcourse) does not exist because this is a book and not a real package. It still needs to be triggered by `devtools::test()`